### PR TITLE
Add sploosh text

### PR DESCRIPTION
**Pull request by cristeahub**

### CODE REVIEW

File: res/layers.json [hunk 1]
```diff
@@ -435,6 +435,26 @@
...
+    "displayName": "Sploosh Text",
+    "startFrame": 3998,
+    "endFrame": 4322,
+    "color": "#a0cc0f",
+    "config": {
+      "title": "Sploosh Effect",
+      "body": [
+        "- organic",
+        "- needed for human reproduction",
+        "- tested in humans"
```
Comment (stianjensen) [inline res/layers.json:448]: tested on humans?

Comment (cristeahub) [inline res/layers.json:448]: no spelling error here

Comment (stianjensen) [inline res/layers.json:448]: :sunglasses:

Comment (cristeahub) [inline res/layers.json]: 8 = :fist:  = = D :sweat_drops:

**Pull request merged**